src/pf-modal/index.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
+    <link rel="stylesheet" href="/dist/css/patternfly-additions.css">
+    <link rel="stylesheet" href="/dist/css/patternfly-webcomponents.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="/dist/js/patternfly.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+      <h1>Modals</h1>
```

Comment:
Please disregard that comment. I thought I was looking at the pf-modal-header. It's difficult to read when GitHub injects all these comments between lines of code.